docs/api/cozy-client.md

```diff
@@ -2349,6 +2357,34 @@ Fetch and build an URL to open a note.
 | client | <code>object</code> | CozyClient instance |
 | file | <code>object</code> | io.cozy.file object |
 
+<a name="saveTimeSerie"></a>
+
+## saveTimeSerie
```

Comment:
Time series has an S at the end even in singular.

Reply:
Good point, fixed with 58f70b7